handbook/engineering/2021_org.md

```diff
@@ -1,46 +1,32 @@
-# 2021 Q1 Organizational Plan
+# 2021 Engineering Organization Plan
 
 Defining a future vision for the organization is necessary to effectively grow the engineering team. It is understood that all plans change, and this document will be kept up to date with these changes. This is our written plan to align the team and clarify hiring and growth expectations.
 
+[2021 headcount plan](https://docs.google.com/spreadsheets/d/1UG4aLB5S8yOSOByia2EQgpZkoPxG5eVfqQqgs1f71Vw/edit#gid=2097845084)
```

Comment:
Are the years at the top of the spreadsheet correct? The earliest is Q3 2021 which is a a year away :)

Reply:
I guess it's financial years in which case it makes sense :)